apps/src/music/views/MusicView.jsx

```diff
@@ -110,6 +122,11 @@ class UnconnectedMusicView extends React.Component {
     this.state = {
       showingVideo: true,
     };
+
+    if (props.appOptions.channel) {
```

Comment:
once we move to the new system we won't care about `channel` in app options anymore (and may not pass it down to `MusicView`). Should we just always say Music Lab is project backed?

Reply:
Oh good call. Yeah, it's probably safe to assume that music lab will always be channel backed since we assume we're getting a channel ID from somewhere